backend/assignments/1/src/main/java/org/example/Main.java

```
@@ -0,0 +1,176 @@
+import org.example.logging.Logger;
+import org.example.operations.CoinOperation;
+import org.example.operations.TraderOperation;
+import org.example.userutility.UserInputHandler;
+import org.example.userutility.UserUtilityMenu;
+
+import java.io.*;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+public class Main {
+    private static final Logger loggerObj = Logger.getLoggerObject();
+    private static final Path TRANSACTION_JSON_FILE_PATH = Path.of(
+            "src/test/resources/test_transaction_1.json");
+
+    public static ArrayList<String[]> parseCSV(Path fileCsvPath) {
+        Logger loggerObj = Logger.getLoggerObject();
+        ArrayList<String[]> csvList = new ArrayList<>();
+        try {
+            Reader csvReader = new FileReader(String.valueOf(fileCsvPath));
+            CSVFormat csvFormat = CSVFormat.DEFAULT.builder().build();
+            Iterable<CSVRecord> csvRecords = csvFormat.parse(csvReader);
+            boolean skipHeader = true;
+            for (CSVRecord rec : csvRecords) {
+                if (skipHeader) {
+                    skipHeader = false;
+                    continue;
+                }
+                csvList.add(rec.values());
+            }
+        } catch (IOException e) {
+            loggerObj.errorLog(e.getMessage(), e);
+        }
+        return csvList;
+    }
+
+    public static JsonNode parseJsonFile(String fileStringPath) {
+        JsonNode jsonTransactions = null;
+        Logger loggerObj = Logger.getLoggerObject();
+        try {
+            File jsonFile = new File(fileStringPath);
+            ObjectMapper mapper = new ObjectMapper();
+            jsonTransactions = mapper.readValue(jsonFile, JsonNode.class);
+        } catch (IOException e) {
+            loggerObj.errorLog(e.getMessage(), e);
+        }
+        return jsonTransactions;
```

Lets not return `null` here. Lets throw an exception and catch it.